### PR TITLE
build: funds localhost env. accounts

### PR DESCRIPTION
**Pull request by AuHau**
This PR adds funding of the TestToken to development accounts

### CODE REVIEW

File: deploy/token.js [hunk 1]
```diff
@@ -1,6 +1,17 @@
-module.exports = async ({ deployments, getNamedAccounts }) => {
+const MINTED_TOKENS = 1_000_000_000
+
+module.exports = async ({ deployments, getNamedAccounts, getUnnamedAccounts }) => {
   const { deployer } = await getNamedAccounts()
-  await deployments.deploy("TestToken", { from: deployer })
+  const tokenDeployment = await deployments.deploy("TestToken", { from: deployer })
+  const token = await hre.ethers.getContractAt("TestToken", tokenDeployment.address)
+
+  const accounts = await getUnnamedAccounts()
```
Comment (markspanbroek) [inline deploy/token.js:8]: Looks like it doesn't mint test tokens for named accounts (account #0 in this case). Is that on purpose? I was a bit surprised that account #0 didn't get any coins.

**Review by markspanbroek — APPROVED** [review]
Looks good!

File: deploy/token.js (hunk 1 above)
Comment (AuHau) [reply deploy/token.js:8]: Yeah it surprised me as well when I was testing it yesterday 😅 We can also mint for it, I don't really mind. Maybe though as it is deployer we should not really use it for testing?

Comment (AuHau) [reply deploy/token.js:8]: I have added also the named accouns in order to prevent possible confusions ;-)

**Pull request merged**